src/design/demo/favicon-svg.twig

```diff
@@ -0,0 +1,15 @@
+<style>
+  @media (prefers-color-scheme: light) {
+    #favicon-demoWrapper {
+      background-color: white;
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    #favicon-demoWrapper {
+      background-color: black;
+    }
+  }
```

Comment:
Can we make the icon preview smaller so it's easier to tell how it will look in a tab?